docs/standard/base-types/standard-numeric-format-strings.md

```diff
@@ -95,6 +96,14 @@ The following sections provide detailed information about each of the standard n
 
 <a name="CFormatString"></a>
```

Comment:
Line 97 needs to be placed just above new line 107 since it supplies navigation to that section.